deploy.sh

```diff
@@ -154,6 +154,14 @@ if [[ $deploy_process_type != "scheduledtasks" && ( -z "$ECS_SERVICE_TASK_PROCES
 			}"
 		)
 
+		# DEPLOY_WEBHOOK_URL="https://hook.us1.make.com/dt8elcq7atff1851avee324lel05vcgm?apikey=a1b2c3d4&cluster={{CLUSTER}}&service={{SERVICE}}&repository={{REPOSITORY}}"
+        if [[ ! -z "$DEPLOY_WEBHOOK_URL" ]]; then
```

Comment:
Looks like this `if` is within the previous one. Newrelic and custom deployments should not be related.